app/models/qdm/tacoma/valueset.rb

```diff
@@ -1,19 +1,17 @@
-module QDM
+module Tacoma
   # ValueSet represents a collection of Concepts, used by the Measures to specify a set of codes for a particular topic
   class ValueSet
     include Mongoid::Document
     field :oid, type: String
     field :display_name, type: String
     field :version, type: String
-    field :user_id, type: String # Eventually we need to delete this from bundles when exporting
-    field :categories, type: Hash
 
     index(oid: 1)
```

Comment:
Doesn't match hash style of other index calls.

Reply:
Fixed.